frontend/src/lib/WidgetStateManager.ts

```diff
@@ -241,6 +242,21 @@ export class WidgetStateManager {
     }
   }
 
+  /**
+   * Sets the trigger value for the given widget ID to true, sends a rerunScript message
```

Comment:
nit: The comment here might need a slight adaption for the string trigger version.